ext/date/tests/bug74639.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Bug #74639 Cloning DatePeriod leads to segfault
+--FILE--
+<?php
+
+$start = new DateTime('2017-05-22 09:00:00', new \DateTimeZone('Europe/Berlin'));
+$end = new DateTime('2017-08-24 18:00:00', new \DateTimeZone('Europe/Berlin'));
```

Comment:
Instead of hard coding the timezone here, I would have put that in an ``--INI--`` setting and used UTC.